backend/application/advertisement.ts

```diff
@@ -15,12 +14,72 @@ export class AdvertisementApplicationService {
           imageUrl,
           author,
           startFrom,
-          endAt
-        }
+          endAt,
+        },
       })
       return advertisement
     } catch (error) {
-      throw new Error(`AdvertisementApplicationService: create advertisement error: ${error}`)
+      throw new Error(
+        `AdvertisementApplicationService: create advertisement error: ${error}`,
+      )
+    }
+  }
+  async getAdvertisement(id: string) {
+    try {
+      const advertisement = await prisma.advertisement.findUnique({
+        where: { id },
+      })
+      return advertisement
+    } catch (error) {
+      throw new Error(
+        `AdvertisementApplicationService: get advertisement error: ${error}`,
+      )
+    }
+  }
+  async getAdvertisements() {
+    try {
+      const advertisement = await prisma.advertisement.findMany({
+        where: {
+          startFrom: {
+            lte: new Date(), // 現在の日付以前の開始日時
+          },
+        },
+      })
+      return advertisement
```

Comment:
```suggestion
      const advertisements= await prisma.advertisement.findMany({
        where: {
          startFrom: {
            lte: new Date(), // 現在の日付以前の開始日時
          },
        },
      })
      return advertisements
```

実態と命名を揃えましょう！
findManyなので入ってくるのは配列で複数形の英語で示しましょう！